packages/web-components/src/biometric-card/biometric-card.ts

```diff
@@ -0,0 +1,63 @@
+import { FASTElement, attr, observable, nullableNumberConverter } from '@microsoft/fast-element';
+import { formValues } from '@open-wc/form-helpers';
+import { densityItemContainer } from '@adaptive-web/adaptive-ui/reference';
+import type { BiometricCardCallback, BiometricCardFormData } from './biometric-card.options.js';
+
+export class BiometricCard extends FASTElement {
+  /**
+   * The placeholder content that gets passed to the internal mobile number text field.
+   */
+  @attr
+  public placeholder: string = '';
+
+  /**
+   * The pattern attribute gets passed to the internal mobile number text field.
+   * @remarks
+   * This attribute matches the platform standard {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern | pattern} attribute on input elements.
+   */
+  @attr
+  public pattern!: string;
+
+  /**
+   * The minlength attribute gets passed to the internal mobile number text field.
+   * @remarks
+   * This attribute matches the platform standard {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength | minlength} attribute on input elements.
+   */
+  @attr({ converter: nullableNumberConverter })
+  public minlength!: number;
+
+  /**
+   * The maxlength attribute gets passed to the internal mobile number text field.
+   * @remarks
+   * This attribute matches the platform standard {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength | maxlength} attribute on input elements.
+   */
+  @attr({ converter: nullableNumberConverter })
+  public maxlength!: number;
+
+  @observable
+  public callback!: BiometricCardCallback;
+
+  /**
+   * @internal
+   */
+  @observable
+  public form!: HTMLFormElement;
+
+  /**
+   * @internal
+   */
+  public handleSubmit(): void {
+    const data = formValues(this.form) as BiometricCardFormData;
+
+    this.callback?.(data);
+    this.$emit('biometric-request', data);
```

Comment:
What are your thoughts on event bubbling or not?  I've been tending to turn bubbling off as a micro-optimization in cases where I figure devs will tend to hook listeners directly to the component anyway as safe to turn it on later if there is a need.

Reply:
What's the optimization benefit to turning it off? I'll admit I haven't dug too deep into configuring events beyond passing the name and event detail, so any help here would be appreciated.

Reply:
I assume that there is more work going on in the background to handle a bubbling event vs. a non-bubbling, but I haven't tested to see what impact it actually has.  To control bubbling you pass it as an option `this.$emit("someEvent", eventDetail, {bubbles: false});`